pywwt/layers.py

```diff
@@ -54,10 +57,19 @@
 
 VALID_STRETCHES = ['linear', 'log', 'power', 'sqrt', 'histeq']
 
+# Save string types for validating ISOT strings in time series tables
+if sys.version_info[0] == 2:
+    STR_TYPE = basestring
+    NP_STR_TYPE = np.string_
+else:
+    STR_TYPE = str
+    NP_STR_TYPE = np.unicode_
+
 # The following are columns that we add dynamically and internally, so we need
 # to make sure they have unique names that won't clash with existing columns
 SIZE_COLUMN_NAME = str(uuid.uuid4())
 CMAP_COLUMN_NAME = str(uuid.uuid4())
+TIME_COLUMN_NAME = str(uuid.uuid4())
```

Comment:
This isn't specific to this PR, but shouldn't we hardcode these to constant values? Generating them on-the-fly just introduces a wildly obscure source of irreproducibility.